### PR TITLE
Add instructions to updating a model

### PR DESCRIPTION
**Pull request by aefhm**
`update` is a bit finicky.

Added clarifications on use.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,6 +93,9 @@ modelActions.save('posts', 42, {
   title: 'Update post'
 })
 ```
+* Note that `update` creates a new Brainstem model and will not make a request if the model is [invalid](https://github.com/mavenlink/brainstem-redux/blob/master/lib/actions/model.js#L53-L58)
+* Adding a reject handler addresses this locally
+* Note that `update` will override optional attributes, so you will need the original model's attributes
```
Comment (juanca) [inline README.md]: Whaddya mean?

File: README.md [hunk 2]
```diff
@@ -93,6 +93,9 @@ modelActions.save('posts', 42, {
   title: 'Update post'
 })
 ```
+* Note that `update` creates a new Brainstem model and will not make a request if the model is [invalid](https://github.com/mavenlink/brainstem-redux/blob/master/lib/actions/model.js#L53-L58)
+* Adding a reject handler addresses this locally
```
Comment (juanca) [inline README.md]: Should this actually go on the previous line. `*` seems like different bullet points.

Comment (aefhm) [inline README.md]: Done.

File: README.md (hunk 1 above)
Comment (aefhm) [inline README.md]: Removed. This was due to my not understanding how Backbone model save is a good PUT request.

Comment (juanca) [inline README.md]: Yeah, though, at some point we'll want to move towards PATCH updates. This is a good thing to keep in mind for when those efforts start rolling!

**Pull request merged**